test/models/playlist_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class PlaylistTest < ActiveSupport::TestCase
+  test "invalid if name is blank" do
+    playlist = Playlist.new(name: "", energy: 35, danceability: 56, genre: "genre1")
+    playlist.valid?
+    assert_not playlist.errors[:name].empty?
```

Comment:
Honnêtement tu peux te contenter de vérifier que `valid?` est faux:

```suggestion
    assert_not playlist.valid?
```

Franchement je chipote, mais c'est plus clair de tester que ce n'est pas valide plutôt que la présence d'une erreur sur name (même si in-fine c'est équivalent, et même ton test est plus précis) + en pratique on ne va pas autant dans le détail.